prisma/schema.prisma

```diff
@@ -120,6 +147,7 @@ model Project {
   stack            ProjectStack @default(OTHERS)
   userId           String
   user             User         @relation(fields: [userId], references: [id], onDelete: Cascade)
+  isFeature        Boolean @default(false)
```

Comment:
![image](https://github.com/user-attachments/assets/1901c160-9103-4ba3-af97-d3e8c2f78454)

there was a "Feature this project" in the figma, so i added this field in the schema. If the isFeature is set to true then the project will show up other it will hide under show more button